tests/unittest/_torch/visual_gen/test_profiler.py

```
@@ -0,0 +1,88 @@
+import pytest
+import torch
+
+from tensorrt_llm._torch.visual_gen.pipeline import (
+    PROFILE_START_STOP_ENV_VAR_NAME,
+    PROFILE_TRACE_ENV_VAR_NAME,
+    BasePipeline,
+)
+
+
+def test_setup_torch_profiler(monkeypatch: pytest.MonkeyPatch) -> None:
+    monkeypatch.setenv(PROFILE_TRACE_ENV_VAR_NAME, "/tmp/visual-gen-trace.json")
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/visual-gen-trace.json"

(S108)